parachain-template/node/Cargo.toml

```diff
@@ -21,6 +21,7 @@ path = "src/main.rs"
 
 [features]
 runtime-benchmarks = ["parachain-template-runtime/runtime-benchmarks"]
+try-runtime = []
```

Comment:
```suggestion
try-runtime = ["parachain-template-runtime/try-runtime"]
```

Reply:
I doubt if this is needed, because of cargo feature leaking issues, otherwise it wouldn't have compiled, but I will apply your commit anyway.